presto-docs/src/main/sphinx/functions/sysinfo.rst

```diff
@@ -0,0 +1,14 @@
+============================
+System Information Functions
+============================
+
+These functions provide standard SQL compliance.
```

Comment:
It does not seem to be main reason for these functions existence.
Maybe sth like `Functions providing information about query execution environment`?

Reply:
That makes more sense.